scripts/tests/test_zkai_proof_pressure_scaling_claim_pack_gate.py

```
@@ -0,0 +1,195 @@
+        payload = self.strip_mutation_summary(self.payload)
+        payload["source_artifacts"][0]["sha256"] = "0" * 64
+        self.assert_rejects(payload, "source digest drift")
+
+    def test_write_outputs_round_trip(self) -> None:
+        gate.validate_payload(self.payload)
+        with tempfile.TemporaryDirectory(dir=gate.EVIDENCE_DIR) as tmp:
+            root = pathlib.Path(tmp)
+            json_path = root / "claim.json"
+            tsv_path = root / "claim.tsv"
+
+            with self.assertRaisesRegex(gate.ProofPressureScalingClaimPackError, "output path must be"):
```

Ruff (0.15.13)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)